src/components/AddItemForm.tsx

```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-function AddItemForm() {
+interface IProps {
+  handleAdd: (textTask: string) => void;
+}
+
+function AddItemForm(props: IProps) {
+  const { handleAdd } = props;
   const domInputId = 'new-todo-input';
+  const [taskText, setTextTask] = useState<string>('');
```

Comment:
lembre-se que o padrão é o mesmo nome do estado com "set" a frente, aqui seria:
```suggestion
  const [taskText, setTaskText] = useState<string>('');
```